charts/sftpgo/values.yaml

```diff
@@ -96,18 +96,19 @@ podAnnotations: {}
 
 # -- Pod [security context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod).
 # See the [API reference](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context) for details.
-podSecurityContext: {}
-  # fsGroup: 2000
+podSecurityContext:
+  fsGroup: 1000
```

Comment:
Is this a requirement for this change?

Reply:
Yes, this was required to give the container access to the attached volume.  Matches the user configuration from the Dockerfile in the sftpgo primary repo.